### PR TITLE
add xlsx file option by using --xlsx parameter 

### PR DESCRIPTION
**Pull request by aristosgi**
Complete issue #1366

In this pull request i modified the following files:

1) requirments.txt i added pandas to requirments
2) Sherlock.py i added parses .xlsx and create a method to export the result as xlxs file if the user use the --xlsx parameter
3).gitignore add *.xlxs in order not to add result files into github

### CODE REVIEW

**Comment by TheYahya** [comment]
Hey @aristosgi,
Nice idea, I get this error thought when trying to run this:
Screen Shot 2022-06-12 at 11 34 58

**Comment by aristosgi** [comment]
On my computer with out problem

image

image

I use python3 i dont know if this is our problem.

**Comment by TheYahya** [comment]
Are you sure that the `requirements.txt` is updated?

**Comment by aristosgi** [comment]
I have added pandas.The last commit was fail i didnt do anything.

**Comment by aristosgi** [comment]
@TheYahya Hello , i just added openpyxl to requirments.txt . I think that this was your problem.This works on my pc with  python3. If the problem remains let me know.Thanks for your time.

**Comment by TheYahya** [comment]
Hey @aristosgi, This is my python version `Python 3.9.12`
Make sure that an venv or isolated environment like docker and check it's work for you with same `requirements.txt`.
 
Screen Shot 2022-06-17 at 00 58 05

**Comment by aristosgi** [comment]
Hello @TheYahya ,

I managed to isolate the requirments and find out whats wrong.The openpyxl was a lower version.I change it and i tested it on a isolated environment and worked fine.

Thanks for your help and your time.

**Review by TheYahya — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -22,6 +22,9 @@ src/
 # Comma-Separated Values (CSV) Reports
 *.csv
 
+#XLSX Reports
+*.xlsx
```
Comment (aristosgi) [inline .gitignore:26]: add this inorder not to add the results that are exported with --xlsx parameter to github

File: requirements.txt [hunk 2]
```diff
@@ -5,3 +5,5 @@ requests>=2.22.0
 requests-futures>=1.0.0
 stem>=1.8.0 
 torrequest>=0.1.0
+pandas>=1.0.0
```
Comment (aristosgi) [inline requirements.txt:8]: add 2 requirments in order to create xlsx files

File: sherlock/sherlock.py [hunk 3]
```diff
@@ -506,6 +507,10 @@ def main():
                         action="store_true", dest="csv", default=False,
                         help="Create Comma-Separated Values (CSV) File."
                         )
+    parser.add_argument("--xlsx",
```
Comment (aristosgi) [inline sherlock/sherlock.py:510]: add --xlsx argument

File: sherlock/sherlock.py [hunk 4]
```diff
@@ -716,6 +722,35 @@ def main():
                                      response_time_s
                                      ]
                                     )
+        if args.xlsx:
```
Comment (aristosgi) [inline sherlock/sherlock.py:725]: here is the main part of the pull req where i create the xlsx option

**Pull request merged**